script.js

```diff
@@ -46,6 +46,260 @@ function createComponent(templateId, template) {
   );
 }
 
+function insertElement(element, content, place_inserted, image) {
+  if (content == null) {
+    content = "";
+  }
+  let area = document.querySelector(place_inserted);
+  if (area == null) {
+    area = document.getElementsByClassName(place_inserted);
+  }
+  if (image == true) {
+    let image = element.setAttribute("src", content);
+    area.appendChild(image);
+  }
+  let text = element.textContent(content);
+  area.appendChild(text);
+}
+function insertImage(image_element, content, place_inserted){
+  // creates source of image tag and inserts it into certain place
+  // insert alt text
+  if (content == null) {
+    content = ""
+  }
+  let area = document.querySelector(place_inserted);
+  let image = image_element.setAttribute(src, content);
+  area.appendChild(image);
+}
+
+function insertText(text_element, content, place_inserted){
+  if (content == null) {
+    content = ""
+  }
+  let area = document.querySelector(place_inserted);
+  let text = text_element.textContent(content);
+  area.appendChild(text);
+}
+
+function insertTextByClass(text_element, content, place_inserted){
+  if (content == null) {
+    content = ""
+  }
+  let area = document.getElementsByClassName(place_inserted);
+  let text = text_element.textContent(content);
+  area.appendChild(text);
+}
+
+function insertImageByClass(image_element, content, place_inserted){
+  // creates source of image tag and inserts it into certain place
+  // insert alt text
+  if (content == null) {
+    content = ""
+  }
+  let area = document.getElementsByClassName(place_inserted);
+  let image = image_element.setAttribute(src, content);
+  area.appendChild(image);
+}
+
+header_image = document.createElement("img");
```

Comment:
You should always declare a scope and reassignment for your variables either using `const`, `let` or `var`. In this case, since you do reassignment, you should use `let`. 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/let